frontend/src/screens/wallet/send/LnurlPay.tsx

```diff
@@ -14,19 +14,18 @@ export default function LnurlPay() {
   const navigate = useNavigate();
   const { toast } = useToast();
 
-  const lnurlDetails = state?.args?.lnurlDetails as LnUrlPayResponse;
+  const lnAddress = state?.args?.lnAddress as LightningAddress;
   const [amount, setAmount] = React.useState("");
   const [comment, setComment] = React.useState("");
   const [isLoading, setLoading] = React.useState(false);
 
   const onSubmit = async (event: React.FormEvent<HTMLFormElement>) => {
     event.preventDefault();
     try {
+      if (!lnAddress) {
+        throw new Error("no lightning address set");
+      }
       setLoading(true);
-      const lnAddress = new LightningAddress(lnurlDetails.identifier);
-      // this is set instead of calling fetch because
-      // requestInvoice uses lnurlpData.callback
-      lnAddress.lnurlpData = lnurlDetails;
       const invoice = await lnAddress.requestInvoice({
```

Comment:
I tried doing this first but for some weird reason this function was not being passed to this screen via state, but somehow it works for me now 👍